src/models.py

```diff
@@ -78,10 +79,17 @@ def serialize(self):
 
     def set_students(self, student_file):
         try:
+            students = []
             lines = student_file.read()
-            students = lines.split(",")
+            if "," in lines:
```

Comment:
I think it would make more sense if we split in this priority:
1 - commas
2 - newlines
3 - spaces
I think newline delimited files often have trailing spaces on each line, so this would account for that.